packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1798,6 +1806,16 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     }
   }
 
+  // null if no promptRect should be shown.
+  TextRange _currentPromptRectRange;
+
+  @override
+  void showAutocorrectionPromptRect(int start, int end) {
```

Comment:
Why did you create this method instead of handling the range as a parameter only?

Reply:
It's part of the `TextInputClient` interface now.

Reply:
Oh I see, I didn't realize how that worked.  Thanks.